scss/main.scss

```diff
@@ -1535,4 +1535,171 @@ p.cardText {
 
 .hidden {
   visibility: hidden;
+}
+
+.starting-projects-container {
+  transition: all ease .2s;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: min-content;
+  max-width: 1400px;
+  padding: 80px 96px;
+  margin-right: auto;
+  margin-left: auto;
+  .starting-projects-info {
+    transition: all ease .2s;
+    position: relative;
+    text-align: center;
+    max-width: 600px;
+    height: max-content;
+    h2 {
+      transition: all ease .2s;
+      color: $darker-grey;
+      height: min-content;
+      padding-top: 0px;
+      margin-bottom: 16px;
+      font-family: $font-family-base;
+      font-style: normal;
+      font-weight: bold;
+      font-size: 3rem;
+    }
+    p {
+      transition: all ease .2s;
+      margin-bottom: 24px;
+      color: $darker-grey;
```

Comment:
Just a note, a lot of these text styles don't need to be redeclared every time. We're inheriting things like color and font-family correctly from the body.